index.js

```diff
@@ -17,7 +17,10 @@ program
     .description('Show all linked packages in your project')
     .action(function () {
         utils.findSymLinks('./node_modules').then(function (linkedModules) {
-            console.log('Currently linked packages to this one\n');
+            var listMessage = linkedModules.length === 0 ?
+                'Currently no linked packages to this one\n' : 'Currently linked packages to this one\n';
```

Comment:
How about have more outstanding message which differ more than two letters? Which means it's more noticeable. Like: `There are no packages linked to this one`.